src/main/java/io/wispforest/owo/util/ReflectionUtils.java

```diff
@@ -2,13 +2,15 @@
 
 import io.wispforest.owo.registration.annotations.AssignedName;
 import io.wispforest.owo.registration.annotations.IterationIgnored;
+import io.wispforest.owo.registration.reflect.MemoizedEntry;
 import org.jetbrains.annotations.ApiStatus;
 import org.jetbrains.annotations.Nullable;
 
 import java.lang.reflect.*;
 import java.util.Locale;
 import java.util.function.Consumer;
 import java.util.function.Function;
+import java.util.function.Supplier;
```

Comment:
are these imports unused? you didn't add any code that actually uses these classes